### DISCUSSION TITLE
Unread notifications badge

### DISCUSSION THREAD

**Discussion by CvBlixen** [community/community#3971] [Mobile] [answered]
I'm bad at now and then looking through my unread GitHub notifications.

I'm quite certain that a red dot with a number on my GitHub app icon would improve that. 🔴

What do you think? 🤔

PS

To be clear: **Just the number of unread notifications.** Nothing fancy, because I love how you're keeping the app simple. 🙂👍

**Comment by elisealix22** [comment, marked as answer]
Thank you so much for your feedback on the app! We've added your request to our own internal Issues. If you have any other feedback specific to the GitHub Mobile apps, don't hesitate to reach back out!

**Reply by donsizzla93** [reply]
- **[]()**

**Comment by CvBlixen** [comment]
> We've added your request to our own internal Issues.

Awesome! Thank you so much!

🙂🙏

PS

Sorry for the late reply. I blame the lack of badges. 😅

**Comment by jaymegordo** [comment]
Any progress on this? Would LOVE notification red dot on the gh mobile app, would remind me to look at and handle all notifications. Thanks!!
